_sass/_mixins.scss

```diff
@@ -40,4 +40,23 @@
     -webkit-filter: grayscale($value);
     -moz-filter: grayscale($value);
     filter: grayscale($value);
+}
+
+@mixin flex-column-centered {
+	display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+@mixin columns-container {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+}
+
+@mixin column-three {
+	flex-basis: 33%;
+	min-width: 200px;
+	padding: $padding-base-horizontal;
```

Comment:
Looking at analytics it seems like most people are using browsers with flexbox support! Is the general consensus these days that flexbox is mainstream?